src/patternfly/components/Page/page.scss

```diff
@@ -287,6 +297,29 @@
     }
   }
 
+  .pf-c-button.pf-m-opened {
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+    width: var(--pf-c-page__header-tools-group--m-icons--c-button--Width);
+    height: var(--pf-c-page__header-tools-group--m-icons--c-button--Height);
+    padding: unset;
+    background-color: var(--pf-c-page__header-tools-group--m-icons--c-button--BackgroundColor);
```

Comment:
Since `--pf-c-page__header-tools-group--m-icons--c-button--BackgroundColor` is transparent, this can be set on the unmodified button.